x-pack/plugins/lists/server/services/exception_lists/exception_list_client.ts

```diff
@@ -263,6 +276,7 @@ export class ExceptionListClient {
     type,
   }: CreateExceptionListItemOptions): Promise<ExceptionListItemSchema> => {
     const { savedObjectsClient, user } = this;
+    await this.createTrustedAppsList();
```

Comment:
If the list didn't already exist then `createExceptionListItemRoute` won't get here (except in race conditions). I think we want to call `createTrustedAppsList` from a separate route similar to `createEndpointListRoute` and allow these routes to return a failure if the list doesn't exist.

Reply:
Apparently we're not calling `createEndpointListRoute` and instead it's automatically created by the app when rules are made...we may want to rework and standardize these processes.

Reply:
Can you call `createTrustedAppsList` from [this block](https://github.com/elastic/kibana/blob/master/x-pack/plugins/lists/server/routes/create_exception_list_item_route.ts#L58) if the `list_id` is `ENDPOINT_TRUSTED_APPS_LIST_ID`? This way we only attempt to create the trusted apps list if we're adding an item to the trusted apps list and the list doesn't exist.

Reply:
@marshallmain I'd honestly rather the list creation be explicit and leave the behavior of that method the same for both list types (otherwise you get a 404 for exception lists, but auto-list creation for trusted apps list, which is weird and unexpected)... we need to figure this out, but I don't want to hold up this PR for this issue. If it's working, I say let's figure out where the lists should actually be created later. Sounds like we need to have a wider discussion on this, as there's conflicting advice and examples throughout the code. Easy enough to change where the lists are created, once the functionality is in place.